src/promptflow/promptflow/_core/tool_meta_generator.py

```diff
@@ -256,6 +256,7 @@ def generate_python_meta_dict(name, content, source=None):
     return asdict_without_none(generate_python_tool(name, content, source))
 
 
+# Only used in non-code first experience.
```

Comment:
Do we still have this?

Reply:
Still has reference from runtime code
https://msdata.visualstudio.com/Vienna/_git/PromptFlow?path=/src/promptflow-sdk/promptflow/runtime/app.py&version=GBdevs/robbenwang_flow_mapping&line=762&lineEnd=762&lineStartColumn=13&lineEndColumn=57&lineStyle=plain&_a=contents